napari/_qt/qt_viewer.py

```diff
@@ -288,7 +289,8 @@ def _create_canvas(self) -> None:
         theme.connect(on_theme_change)
 
         def disconnect():
-            theme.disconnect(on_theme_change)
+            with suppress(AttributeError):
```

Comment:
Can you maybe add a comment here on why there might be an `AttributeError`

Reply:
I honestly have no idea... i think it has something to do with cleanup order?  Will add a note about it being mysterious

Reply:
lol, ok, nice - a note about it being mysterious is better than no note at all :-)